R/db-odbc-mssql.R

```diff
@@ -101,3 +101,37 @@
   DBI::dbExecute(con, sql)
 }
 
+#' @export
+`db_compute.Microsoft SQL Server` <- function(
+                con,
+                table,
+                sql,
+                temporary = TRUE,
+                unique_indexes = list(),
+                indexes = list(),
+                ...) {
+
+
+  # The user will have to pass the pound sign, or signs, to setup the
+  # temp table. Two pound signs indicate a global temporary table, a
+  # single pound sign indicates a local temporary table
```

Comment:
I looked into that some time back.  The issue I wasn't able to overcome was that the table name retains the double pound, so the R users will either have to know to use the double pound prefix when calling the table via `tbl()` or we will have to somehow continue to prefix the name on all functions that make table references.  I was about to submit an update that returns an error if the table name is not prefixed with a pound.